StaticHelpers/StringHelpers.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection.Metadata;
+
+namespace PhoneBook.StaticHelpers
+{
+    public static class StringHelpers
+    {
+
+        /*"N"---------- number
+          "C"---------- char
+          "(''||'')"--- Or              In dev
+         */
+
+        public static bool Validator(this string str)
+        {
+            string myString = str;
+            int formatLenght = 9;
```

Comment:
This should not be defined on each run of this method. The Number Length is something that will not change during program runtime